lib/index.js

```diff
@@ -8,6 +8,14 @@ const miniget = require('miniget');
 const m3u8stream = require('m3u8stream');
 const parseTime = require('m3u8stream/dist/parse-time');
 
+const current = 'v4.0.5';
+const message = '\x1B[31mWARNING:\x1B[0m ytdl-core is out of date! Update it using "npm i ytdl-core".';
+miniget('https://api.github.com/repos/fent/node-ytdl-core/releases/latest', { headers: { 'User-Agent': 'a/b' } })
```

Comment:
is the `User-Agent` header needed?

Reply:
while I was testing this, I got 403s from the service if it wasn't there. weird, but any valid user-agent fixed it.
Using node-fetch seems to solve that entirely, but i didnt want to add another dependency.